app/models/role_invite.py

```diff
@@ -31,7 +31,11 @@ class RoleInvite(SoftDeletionModel):
     role = db.relationship("Role")
 
     hash = db.Column(db.String)
+<<<<<<< HEAD
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible SCM diff in code](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4778007)